### PR TITLE
feat(sparkle): meru builder layout

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

sparkle changes required for https://github.com/dust-tt/dust/pull/4144

Mar-05-2024 14-17-22

## Risk

none: only used in the unmerged branch linked above

### CODE REVIEW

File: sparkle/src/components/BuilderLayout.tsx [hunk 1]
```diff
@@ -8,48 +8,56 @@ import { Button } from "./Button";
...
 }
 
-export function BuilderLayout({ leftPanel, rightPanel }: BuilderLayoutProps) {
-  const [isRightPanelOpen, setIsRightPanelOpen] = useState(true);
-
-  const toggleRightPanel = () => {
-    setIsRightPanelOpen(!isRightPanelOpen);
-  };
-
+export function BuilderLayout({
```
Comment (fontanierh) [inline sparkle/src/components/BuilderLayout.tsx:15]: - made it non-managed
- removed the background colors
- made it work on mobile (no right panel / preview + no button)
- tweaked the sizes

Comment (philipperolet) [reply sparkle/src/components/BuilderLayout.tsx:15]: :+1: 
But in the stories.tsx can you put background colors & lorem ipsum content so the sparkle doc shows better if the preview hides or pushes content, and/or how it's transitioned if it is?

**Review by philipperolet — APPROVED** [review]
:+1: 
nit to improve the sparkle story (on the gif ambiguity about what happens exactly because all white)

**Comment by fontanierh** [comment]
fair @philipperolet ! fixed.

**Pull request merged**